### ISSUE TITLE
analysis_options_package.yaml cannot be found

### ISSUE THREAD

**Issue by Abhilash-Chandran** [passsy/dart-lint#16]
I am using VSCode and I added `lint:^1.2.0` as a `dev_dependencies`.

Now trying to import `analysis_options_package.yaml` as shown below breaks the `flutter analyze`.

`include: package:lint/analysis_options_package.yaml`

Following warning is raise by `flutter analyze`.

```
warning - The include file package:lint/analysis_options_package.yaml in
       C:\my\local\path\to\analysis_options.yaml cannot be found -
       analysis_options.yaml:1:11 - include_file_not_found
```
Should be added to normal `dependencies` instead `dev_dependecies`. In my case that didn't help either. Not sure if I am missing any step.

**Comment by kuhnroyal** [comment]
I have been using the lint package for a while and now I am suddenly getting the same problem in a new project.

**Comment by passsy** [comment]
If you have a project with multiple modules you might run into this bug: https://github.com/dart-lang/sdk/issues/42910

When it happens for a single module this [bug](https://github.com/Dart-Code/Dart-Code/issues/1130) was fixed in Dart 2.10. Try to upgrade if possible.

**Comment by kuhnroyal** [comment]
Yea I just found and commented on this issue. Thanks!

**Comment by Abhilash-Chandran** [comment]
Thanks for the feed back @passsy. I checked even the closed issues list before opening this. Hope its not a repeated issue. Should I close this or leave it open until the main issue is resolved..?

**Comment by passsy** [comment]
Close it until the issue is resolved on your side

**Comment by passsy** [comment]
The workaround https://github.com/dart-lang/sdk/issues/42910#issuecomment-851943584, adding `analysis_options.yaml` in *every* project is working. Including example projects
